### DISCUSSION TITLE
onboard led to be used as an activity light

### DISCUSSION THREAD

**Discussion by extremeroom7u7** [risinek/esp32-wifi-penetration-tool#82] [Ideas]
it would be cool for the onboard led to be used as an activity light indicator for packed sending when doing attack

**Comment by risinek** [comment]
that would bind the project to a specific board. the way it is designed now it should work on any ESP32 module regardless what board it is integrated to.

**Reply by extremeroom7u7** [reply]
Oh well maybe there's a way around it?

**Reply by risinek** [reply]
around what? it's definitely possible to implement, but I don't want to do it in this project. It should be easy if the LED on your board is controllable from ESP32. Look for the board datasheet and then just flip the relevant pin when the packet is being sent.

**Reply by extremeroom7u7** [reply]
Alright. How do I edit the bin file so I can do it myself

**Reply by risinek** [reply]
you don't edit bin files. you edit source code.. see readme for build instructions and read ESP-IDF docs to understand the basics

**Comment by extremeroom7u7** [comment]
Okay! Thank u
